src/lists.js

```diff
@@ -687,9 +687,9 @@ List.prototype.reshape = function (dimensions) {
     // answer a new list formatted to fit the given dimensions.
     // truncate excess elements, if any.
     // pad with (repetitions of) existing elements
-    var src = this.ravel().itemsArray(),
-        size = dimensions.isEmpty() ? 0
-            : dimensions.itemsArray().reduce((a, b) => a * b),
+    var src = this.ravel().itemsArray();
```

Comment:
Jens’ style would be to do this:
```suggestion
    var size, src = this.ravel().itemsArray();
```

then remove the next var

Reply:
OK, done, except there were two more variables to declare so I moved those up too.

Sorry for all the confusing reverts; somehow Github Desktop seemed to be saying that I'd pushed this change right into the master branch.  But all is well.